.eslintignore

```diff
@@ -1,3 +1,4 @@
 /coverage
 /third_party
 /public/js/gulliver.js
+/public/js/ligthouse-chart.js
```

Comment:
I certainly have a big problem spelling lighthouse :(

The problem is that 'google' is not defined  no-undef, because is imported in the page and not in this file, how can I solve that?